collaboration-engine-test/collaboration-engine-integration-test/pom.xml

```diff
@@ -21,6 +21,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>collaboration-engine</artifactId>
+            <version>7.0-SNAPSHOT</version>
```

Comment:
Any reason not to use `${project.version}`?

Reply:
it is strange to add this dependency here, as this dependency should come from `collaboration-engine-test-common`